sessionctx/variable/session.go

```diff
@@ -932,6 +939,27 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		if isAutocommit {
 			s.SetStatusFlag(mysql.ServerStatusInTrans, false)
 		}
+	case AutoIncrementIncrement:
+		// AutoIncrementIncrement is valid in [1, 65535].
+		temp := tidbOptPositiveInt32(val, DefAutoIncrementIncrement)
+		if temp <= 0 {
+			s.AutoIncrementIncrement = 1
+		} else if temp > math.MaxUint16 {
+			s.AutoIncrementIncrement = math.MaxUint16
+		} else {
+			s.AutoIncrementIncrement = temp
+		}
+
+	case AutoIncrementOffset:
+		// AutoIncrementOffset is valid in [1, 65535].
+		temp := tidbOptPositiveInt32(val, DefAutoIncrementOffset)
+		if temp <= 0 {
+			s.AutoIncrementOffset = 1
+		} else if temp > math.MaxUint16 {
+			s.AutoIncrementOffset = math.MaxUint16
+		} else {
+			s.AutoIncrementOffset = temp
+		}
```

Comment:
Can we use a function to wrap them?

Reply:
it's ok here.

Reply:
Line 944-951 and Line 953 to 961 are equal, we shouldn't let the code duplicate

Reply:
addressed